lib/drivers/medtronic600/medtronic600Simulator.js

```diff
@@ -475,9 +475,6 @@ class Medtronic600Simulator {
               break;
             }
           }
-          if (suppressedBasal == null) {
```

Comment:
Just checking: So it's ok if the previous suppressed basal is not found?

Reply:
Yep, because if the suppressed basal is missing, it won't generate the suspend, which means that we have to rely on a previous upload, or just don't upload it.
Take this scenario:
A schedule basal of 3 hours happens, then there's a suspend 2 hours in for 30 minutes. That works fine if all the data is there.
However, if we started reading the data before the schedule event (say an hour into the scheduled basal), we'll get the suspend event, but NOT the underlying suppressed information. In that instance, we don't show the scheduled basal, OR the suspend. 😄